### DISCUSSION TITLE
Metviewer is done or have a new address?

### DISCUSSION THREAD

**Discussion by Ho-ChunHuang-NOAA** [dtcenter/METplus#1785] [Plot Generation] [answered]
I can access this site yesterday but not today

https://metviewer.nws.noaa.gov/metviewer1.jsp

It becomes non accessible after I put in usrname and passwd.

"Internal Server Error
The server encountered an internal error or misconfiguration and was unable to complete your request.

Please contact the server administrator at root@localhost to inform them of the time this error occurred, and the actions you performed just before this error.

More information about this error may be available in the server error log."

**Comment by malloryprow** [comment, marked as answer]
@Ho-ChunHuang-NOAA I was able to access it just now.

**Comment by Ho-ChunHuang-NOAA** [comment]
Hi, Mallory:

Thanks.  I switched to FireFox and I can connect as well.  But I still have
problems using Chrome.

Please close this ticket, it appears to be a local connection issue.

Ho-Chun Huang, Ph.D.

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958
